src/client/client.c

```diff
@@ -39,10 +38,8 @@ int main(int argc, char** argv) {
   }
 
   struct sockaddr_in server;
-  server.sin_family = AF_INET;
-  server.sin_port = htons(ip_port.port);
-  server.sin_addr.s_addr = *((unsigned long*)hostname->h_addr);
-
+  set_sockaddr_in(&server, AF_INET, ip_port.port,
+  	*((unsigned long*)hostname->h_addr));
```

Comment:
We can't avoid this, because in 'server' we pass this
![image](https://user-images.githubusercontent.com/61005445/84870582-2c178080-b088-11ea-9b59-302c5dc67234.png)